quiche/http2/decoder/http2_frame_decoder_test.cc

```diff
@@ -108,7 +108,7 @@ class Http2FrameDecoderTest : public RandomDecoderTest {
 
   // Decode one frame's payload and confirm that the listener recorded the
   // expected FrameParts instance, and only one FrameParts instance. The
-  // payload will be decoded several times with different partitionings
+  // payload will be decoded several times with different partitions
```

Comment:
it's possible there's more meaning entailed here, but I _think_ that this change works